modules/images-other-jenkins-agent-images.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-images_{context}"]
+= Jenkins agent images
+
+The {product-title} Jenkins agent images come in two varieties:
+
+*RHEL 7 Based Images*
+
+RHEL 7 images are available through the Red Hat Registry:
+
+----
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-base-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-maven-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-nodejs-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-agent-maven-35-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-agent-nodejs-8-rhel7
```

Comment:
From 4.x, jenkins and agent images are integrated to payload, For released image, should be quay.io, not get from registry.redhat.io

Reply:
See here
$ oc adm release info --pullspecs registry.svc.ci.openshift.org/ocp/release:4.1.0-rc.7 | grep  jenkins 
  jenkins                                       quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2600aaa3b380a0be3a2b8c67229209968625d4bfff9579326fd3bee811e951bc
  jenkins-agent-maven                           quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:066b177b748ea97741521b716779d9ca6daac87bd8ecd58c5607da3737924436
  jenkins-agent-nodejs                          quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fd083d0e43161f38821ca3de1e16ed10906f47c3790cd2f2389ce555a8b4876a